javascripts/background_scripts/modal_button.js

```diff
@@ -1,64 +1,141 @@
 /**
- * @fileOverview This file initializes the modal button. This includes
- * initializing the button's state on browser start and interfacing with
- * the reading_process.js so the links will toggle their injection.
- * Changes to the modal button are also made by the popup.html
- * page which provides some controls and links for the extension.
+ * @fileOverview This file initializes the modal button (browser action).
+ * If the user is inputing in a seamless-posting form, the modal button
+ * will become green. In other cases, it indicates whether content injection
+ * is enabled: red for enabled, gray for disabled.
```

Comment:
Sorry I forgot to update those comments. The icon has already changed to gray/green.